src/legacy/core_plugins/metrics/public/components/vis_types/metric/series.test.tsx

```diff
@@ -18,16 +18,15 @@
  */
 
 import React from 'react';
+// @ts-ignore
```

Comment:
What is the added `ts-ignore` for here?

Reply:
Rolled back to a js file temporarily (until we deangularize vis types)